react-search/my-app/src/SearchBar.tsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import './SearchBar.css';
+
+type Props = {
+  onSearchChange: (event: React.ChangeEvent<HTMLInputElement>) => void;
```

Comment:
You could hide the event by passing the text as the parameter instead of the event:
```
type Props = {
  onSearchChange: (value: string) => void;
}

// below
return <input type="text" onChange={(e) => onSearchChange(e.target.value)} />;
```

As you can see, I think your code is cleaner and easier to work with if you extract the information out of the event in the child components. That's a good pattern to always use.